kotlinx-coroutines-core/common/src/channels/Channels.common.kt

```diff
@@ -3,7 +3,7 @@
  */
 @file:JvmMultifileClass
 @file:JvmName("ChannelsKt")
-@file:Suppress("DEPRECATION_ERROR")
+@file:Suppress("DEPRECATION_ERROR", "DEPRECATION")
```

Comment:
I don't like this `Suppress`, it could obscure some real issues eventually. I checked, and the compiler seems pleased if we just add the `Suppress` to each deprecated method separately: https://gist.github.com/dkhalanskyjb/de271eff5aa3286fd680769e080696f8